### PR TITLE
Update the directory names of UFS SRW App renamed to meet NCO standards

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES: 
- The directory names of the UFS SRW App are updated to meet the NCO standards. 
- The workflow environment files in the 'ufs-srweather-app/env' directory are moved to the 'modulefiles/env' directory.

## TESTS CONDUCTED:
WE2E Test  
- grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
- grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson_mynn_lam3km
- MET_ensemble_verification

## ISSUE: 
Fixes issue mentioned in #678.

### CODE REVIEW

**Review by RatkoVasic-NOAA — APPROVED** [review]
Looks good to me.

**Comment by gsketefian** [comment]
@chan-hoo I think you can just delete the `docs` directory since all documentation is now in the `ufs-srweather-app` repo.

**Comment by gsketefian** [comment]
@chan-hoo There is a comment block at the top of `ush/config_defaults.sh` that references the latest NCO standards document (lines 15-32).  Do you mind updating that from the 2019 to the 2022 standard?  Thanks.

**Comment by chan-hoo** [comment]
@gsketefian, updated.

File: ush/launch_FV3LAM_wflow.sh [hunk 1]
```diff
@@ -116,7 +116,7 @@ expt_name="${EXPT_SUBDIR}"
 #
 #-----------------------------------------------------------------------
 #
-env_fp="${SR_WX_APP_TOP_DIR}/env/${WFLOW_ENV_FN}"
+env_fp="${HOMErrfs}/modulefiles/env/${WFLOW_ENV_FN}"
 module purge
```
Comment (JeffBeck-NOAA) [inline ush/launch_FV3LAM_wflow.sh:120]: I think we want to keep this as what's on the left-hand side, correct?  Also, it should really be ${SRW_APP_TOP_DIR} (not SR_WX_APP...), but that's beyond the scope of this PR!

**Review by JeffBeck-NOAA — CHANGES_REQUESTED** [review]

**Pull request closed without merging**